### PR TITLE
Add the URL Pattern Standard

### PR DESCRIPTION
**Pull request by domenic**
See https://github.com/whatwg/sg/issues/215.

### CODE REVIEW

**Comment by domenic** [comment]
@jeremyroman, @sisidovski, @wanderview, please check that your details are correct, in terms of which names and emails you want listed. The information will be used to generate bibliography entries which show up when other specs reference you, and I will use it as a template when adding the acknowledgments section to the spec..

In particular, @sisidovski if you would like your kanji name listed, we can do that. Usually we do it with the romaji name in parentheses, like [here](https://streams.spec.whatwg.org/#acks).

File: db.json [hunk 1]
```diff
@@ -529,6 +529,31 @@
...
+            {
+              "name": "Ben Kelly",
+              "email": "wanderview@chromium.org"
+            },
+            {
+              "name": "Jeremy Roman",
+              "email": "jbroman@chromium.org"
+            },
+            {
+              "name": "Shunya Shishido",
```
Comment (sisidovski) [inline db.json:547]: 宍戸俊哉 (Shunya Shishido) is preferable, thank you.

**Review by annevk — APPROVED** [review]
Approving non-name details (and also to make merging this no longer blocked on the SG).

**Comment by wanderview** [comment]
LGTM

**Comment by jeremyroman** [comment]
My name and email address are accurate, LGTM.

**Pull request merged**